ios/scaffold/apple-app-site-association

```diff
@@ -0,0 +1,21 @@
+{
+    "applinks": {
+        "apps": [],
+        "details": [
+            {
+                "appID": "DEV_TEAM_ID.com.mobify.astro",
```

Comment:
I think this might need to be "DEV_TEAM_ID.com.mobify.astro.scaffold" to get replaced by the generator script (not a big deal either way though)